.changeset/curly-lies-flash.md

```diff
@@ -0,0 +1,6 @@
+---
+'@vanilla-extract/css': minor
+'site': minor
```

Comment:
We should not version the site. Can we also add a minor for the affected plugins.

Reply:
Updated with the affected packages, let me know if I missed some